Kanban_bellomsean@gmail.com.md

```diff
@@ -0,0 +1,83 @@
+# Topic: Kanban
+
+**Author**: Sean Bellom
+
+**QAs Total**: 3
+
+---
+
+## Q1: How is _Kanban_ useful in Project Management?
+
+**Difficulty:** `Junior`
+
+**Source:**
+
+https://www.paymoapp.com/blog/what-is-kanban/
+https://www.knowledgehut.com/interview-questions/kanban
+
+**Answer:**
+
+**Kanban** is a Project Management tool that allows you to get a **more visual overview** of the tasks that either need to get done or are complete.
+
+At its core, it consists of a physical or _digital board_ with three columns (_To Do, In Progress, Done_) and tasks listed as story cards. Every card will be pulled from left to right until it gets done. Finally, they will leave the workflow.
+
+**With the help of Kanban, we can do the following;**
+* Organize your work into projects
+* Build project teams
+* Invite people to join the team
+* Share and collaborate on work to do- using online Kanban boards
+
+Its **main goal**? To visualize and maintain the workflow while eliminating waste.
+
+![](https://cfw.paymoapp.com/wp-content/uploads/2020/05/Kanban-method@2x.png)
+
+---
+
+## Q2: Explain how you can link a card together in _Kanban_?
```

Comment:
Hi Sean! Love your work. The only concern I have is the Q2. Looks like it's super specific to the kanbantool software product and won't be relevant to PMStack.Cafe audience. Can I ask you to swap it for another Mid Kanban-related question? Thanks in advance!

Reply:
Hi Alex! Thank you for your review 🙂.

I have replaced the QA2, let me know if this is ok.

Thank you in advance